src/YesSql.Core/Commands/CreateIndexCommand.cs

```diff
@@ -34,19 +38,19 @@ public class CreateIndexCommand : IndexCommand
             {
                 var sql = Inserts(type) + $" {dialect.IdentitySelectString} id";
                 Index.Id = await connection.ExecuteScalarAsync<int>(sql, Index, transaction);
-                await connection.ExecuteAsync($"update [{_tablePrefix}{type.Name}] set DocumentId = @mapid where Id = @Id", new { mapid = Index.GetAddedDocuments().Single().Id, Id = Index.Id }, transaction);
+                await connection.ExecuteAsync($"update {_openQuoteDialect}{_tablePrefix}{type.Name}{_closeQuoteDialect} set DocumentId = @mapid where Id = @Id", new { mapid = Index.GetAddedDocuments().Single().Id, Id = Index.Id }, transaction);
             }
             else
             {
                 var reduceIndex = Index as ReduceIndex;
 
-                var sql = Inserts(type) + $"; {dialect.IdentitySelectString} id";
+                var sql = Inserts(type) + $" {dialect.IdentitySelectString} id";
                 Index.Id = await connection.ExecuteScalarAsync<int>(sql, Index, transaction);
 
                 var bridgeTableName = type.Name + "_" + documentTable;
-                var columnList = $"[{type.Name}Id], [DocumentId]";
+                var columnList = $"{_openQuoteDialect}{type.Name}Id{_closeQuoteDialect}, {_openQuoteDialect}DocumentId{_closeQuoteDialect}";
```

Comment:
Instead of the `openQuote` variable we need to call the dialect with `FormatTable` and `FormatColumn`